### PR TITLE
[stable/jiva]: update jiva charts to 2.12.0 release

### PR DESCRIPTION
**Pull request by shubham14bajpai**
Signed-off-by: shubham <shubham.bajpai@mayadata.io>

### CODE REVIEW

**Review by niladrih — APPROVED** [review]
LGTM

File: deploy/helm/charts/values.yaml [hunk 1]
```diff
@@ -111,7 +111,7 @@ jivaCSIPlugin:
     repository: openebs/jiva-csi
     pullPolicy: IfNotPresent
     # Overrides the image tag whose default is the chart appVersion.
-    tag: 2.11.0
+    tag: 2.12.0
```
Comment (kmova) [inline deploy/helm/charts/values.yaml:114]:
```suggestion
    tag: 2.12.1
```

Comment (kmova) [reply deploy/helm/charts/values.yaml:114]: can the repository name be changed to "openebs/jiva-operator" in the above lines..

File: deploy/helm/charts/values.yaml [hunk 2]
```diff
@@ -25,20 +25,20 @@ jivaOperator:
...
+      tag: 2.12.0
   image:
     # Make sure that registry name end with a '/'.
     # For example : quay.io/ is a correct value here and quay.io is incorrect
     registry:
     repository: openebs/jiva-operator
     pullPolicy: IfNotPresent
     # Overrides the image tag whose default is the chart appVersion.
-    tag: 2.11.0
+    tag: 2.12.0
```
Comment (kmova) [inline deploy/helm/charts/values.yaml:41]:
```suggestion
    tag: 2.12.1
```

File: deploy/helm/charts/values.yaml [hunk 3]
```diff
@@ -25,20 +25,20 @@ jivaOperator:
...
       registry:
       repository: openebs/jiva
-      tag: 2.11.0
+      tag: 2.12.0
   replica:
     image:
       registry:
       repository: openebs/jiva
-      tag: 2.11.0
+      tag: 2.12.0
```
Comment (kmova) [inline deploy/helm/charts/values.yaml:33]:
```suggestion
      tag: 2.12.1
```

File: deploy/helm/charts/values.yaml [hunk 4]
```diff
@@ -25,20 +25,20 @@ jivaOperator:
...
```
Comment (kmova) [inline deploy/helm/charts/values.yaml:28]:
```suggestion
      tag: 2.12.1
```

File: deploy/helm/charts/values.yaml [hunk 5]
```diff
@@ -3,7 +3,7 @@
 # Declare variables to be passed into your templates.
 
 release:
-  version: "2.11.0"
+  version: "2.12.0"
```
Comment (kmova) [inline deploy/helm/charts/values.yaml:6]:
```suggestion
  version: "2.12.1"
```

File: deploy/helm/charts/README.md [hunk 6]
```diff
@@ -182,12 +183,12 @@ helm upgrade openebs-jiva openebs-jiva/jiva -n openebs \
...
 | jivaCSIPlugin.image.registry | string | `nil` | Jiva CSI driver image registry |
 | jivaCSIPlugin.image.repository | string | `"openebs/jiva-csi"` |  Jiva CSI driver image repository |
-| jivaCSIPlugin.image.tag | string | `"2.11.0"` | Jiva CSI driver image tag |
+| jivaCSIPlugin.image.tag | string | `"2.12.0"` | Jiva CSI driver image tag |
 | jivaCSIPlugin.name | string | `"jiva-csi-plugin"` | Jiva CSI driver container name |
 | jivaCSIPlugin.remount | string | `"true"` | Jiva CSI driver remount feature, enabled by default |
 | rbac.create | bool | `true` | Enable RBAC |
 | rbac.pspEnabled | bool | `false` | Enable PodSecurityPolicy |
-| release.version | string | `"2.11.0"` | Openebs Jiva release version |
+| release.version | string | `"2.12.0"` | Openebs Jiva release version |
```
Comment (kmova) [inline deploy/helm/charts/README.md:191]:
```suggestion
| release.version | string | `"2.12.1"` | Openebs Jiva release version |
```

File: deploy/helm/charts/README.md [hunk 7]
```diff
@@ -182,12 +183,12 @@ helm upgrade openebs-jiva openebs-jiva/jiva -n openebs \
 | jivaCSIPlugin.image.pullPolicy | string | `"IfNotPresent"` | Jiva CSI driver image pull policy |
 | jivaCSIPlugin.image.registry | string | `nil` | Jiva CSI driver image registry |
 | jivaCSIPlugin.image.repository | string | `"openebs/jiva-csi"` |  Jiva CSI driver image repository |
-| jivaCSIPlugin.image.tag | string | `"2.11.0"` | Jiva CSI driver image tag |
+| jivaCSIPlugin.image.tag | string | `"2.12.0"` | Jiva CSI driver image tag |
```
Comment (kmova) [inline deploy/helm/charts/README.md:186]:
```suggestion
| jivaCSIPlugin.image.tag | string | `"2.12.1"` | Jiva CSI driver image tag |
```

File: deploy/helm/charts/README.md [hunk 8]
```diff
@@ -166,14 +166,15 @@ helm upgrade openebs-jiva openebs-jiva/jiva -n openebs \
...
+| defaultClass.allowVolumeExpansion | bool | `true` | Enable volume expansion for the Volumes |
 | defaultClass.reclaimPolicy | string | `"Delete"` | Reclaim Policy for the StorageClass |
 | defaultClass.isDefaultClass | bool | `false` | Make jiva csi StorageClass as the default StorageClass |
 | jivaOperator.annotations | object | `{}` | Jiva operator annotations |
 | jivaOperator.componentName | string | `"jiva-operator"` | Jiva operator component name |
 | jivaOperator.image.pullPolicy | string | `"IfNotPresent"` | Jiva operator image pull policy |
 | jivaOperator.image.registry | string | `nil` | Jiva operator image registry |
 | jivaOperator.image.repository | string | `"openebs/jiva-operator"` | Jiva operator image repository |
-| jivaOperator.image.tag | string | `"2.11.0"` |  Jiva operator image tag |
+| jivaOperator.image.tag | string | `"2.12.0"` |  Jiva operator image tag |
```
Comment (kmova) [inline deploy/helm/charts/README.md:177]:
```suggestion
| jivaOperator.image.tag | string | `"2.12.1"` |  Jiva operator image tag |
```

File: deploy/helm/charts/Chart.yaml [hunk 9]
```diff
@@ -4,10 +4,10 @@ description: Jiva-Operator helm chart for Kubernetes
 type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 2.11.1
+version: 2.12.0
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: 2.11.0
+appVersion: 2.12.0
```
Comment (kmova) [inline deploy/helm/charts/Chart.yaml:10]:
```suggestion
appVersion: 2.12.1
```

**Review by prateekpandey14 — APPROVED** [review]

**Pull request merged**